manifests/kfdef/odh-dashboard-kfnbc-test.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  annotations:
+    kfctl.kubeflow.io/force-delete: "false"
+  name: odh-dashboard-kfnbc-test
+  namespace: opendatahub
+spec:
+  applications:
+  - kustomizeConfig:
+    # This includes some common k8s defaults for an ODH Deployment like namespace and operatorGroup
+    # TODO: Review/Refactor this to see if it is required for deploying KFNBC and odh-dashboard
+      repoRef:
+        name: manifests
+        path: odh-common
+    name: odh-common
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: odh-notebook-controller
+    name: odh-notebook-controller
```

Comment:
Should we add the notebook images `kustomizeConfig` here too?

```
  - kustomizeConfig:
      overlays:
      - additional
      repoRef:
        name: manifests
        path: jupyterhub/notebook-images
    name: notebook-images
```

Reply:
Done